lib/core/versions/latest/AnchorFile.ts

```diff
@@ -170,6 +180,7 @@ export default class AnchorFile {
     });
 
     const anchorFileModel = {
+      writerLock: writerLock,
```

Comment:
Nit:
```suggestion
      writerLock,
```

Reply:
Renaming it to `writerLockId` to match the `.getWriterLock(...)` function that is in the `IBlockchain`.